Owncloud iOs Client/AppDelegate.m

```diff
@@ -1700,8 +1682,15 @@ - (void)application:(UIApplication *)application handleEventsForBackgroundURLSes
 
 #pragma mark - Pass Code
 
+- (void)checkIfIsNecessaryShowTouchId {
```

Comment:
The name of the method seems a bit confusing; it's not just checking, it's also showing the TouchId.

What about something like ```showTouchIdIfNeeded``` ?